packages/runtime/src/helpers/files.ts

```diff
@@ -386,82 +336,6 @@ export const getDependenciesOfFile = async (file: string) => {
   return dependencies.files.map((dep) => resolve(dirname(file), dep))
 }
 
-const baseServerReplacements: Array<[string, string]> = [
```

Comment:
We no longer need the server patches her because we handle this in NetlifyNextServer now.